tutorials/hosted_phoenix/hosted_phoenix_openai_tutorial.ipynb

```diff
@@ -0,0 +1,516 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.        dataset_name="test-dataset-2",
</pre><p>Can you make this dataset more realistic?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/4018/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>